samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -2573,6 +2585,38 @@ MFX_IOPATTERN_IN_VIDEO_MEMORY : MFX_IOPATTERN_IN_SYSTEM_MEMORY);
 #endif
     }
 
+    if (pInParams->nAvcTemp)
+    {
+        if (pInParams->EncodeId == MFX_CODEC_HEVC)
+        {
+            int i;
```

Comment:
better to move it in for loop ```for (int i = 0; ...)```

Reply:
Fixed. Thanks.
Updated commit:  3f5bc4f